examples/fabric-admin/commands/pairing/OpenCommissioningWindowCommand.h

```diff
@@ -47,6 +48,10 @@ class OpenCommissioningWindowCommand : public CHIPCommand
                     &mIteration, "Number of PBKDF iterations to use to derive the verifier.  Ignored if 'option' is 0.");
         AddArgument("discriminator", 0, 4096, &mDiscriminator, "Discriminator to use for advertising.  Ignored if 'option' is 0.");
         AddArgument("timeout", 0, UINT16_MAX, &mTimeout, "Time, in seconds, before this command is considered to have timed out.");
+        AddArgument("salt", &mSalt, "Salt payload encoded in hexadecimal. Random salt will be generated if absent");
```

Comment:
This doesn't match the actual code, which requires the salt to be specified in some cases.  I strongly suggest having the docs not be misleading.

Reply:
Added a comment that the corresponding salt needs to be present when the verifier is provided: https://github.com/project-chip/connectedhomeip/pull/34209/commits/da28dd643e7419092f7a073ab2011fda251f2504